package.json

```diff
@@ -95,6 +95,7 @@
     "**/json-schema": "^0.4.0",
     "**/pdfkit/crypto-js": "4.0.0",
     "**/react-syntax-highlighter": "^15.3.1",
+    "**/react-syntax-highlighter/**/highlight.js": "^10.4.1",
```

Comment:
Any context on this?  It doesn't seem to exist on main.

Reply:
It existed when SB was upgraded to 6.5 (#139334) and I thought it may have been related to a type issue. Removed now.